### PR TITLE
`animations`: Add `flutter_animate`

### PR DESCRIPTION
**Pull request by domesticmouse**
## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: animations/lib/main.dart [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import 'dart:io';
 
+import 'package:animations/src/misc/flutter_animate.dart';
```
Comment (parlough) [inline animations/lib/main.dart:7]: Seems you use local imports for the other demos.

Comment (domesticmouse) [reply animations/lib/main.dart]: Whups, I forgot to add it to the `misc.dart` export list. Fixed.

**Review by parlough — APPROVED** [review]
Fun! I have looked at `flutter_animate` only a little bit, but this looks good to me and works well locally.

**Comment by domesticmouse** [comment]
> Fun! I have only looked at `flutter_animate` a little bit, but this looks good to me and works well locally.

Grant Skinner recorded a video about this package. It's in the Flutter Forward playlist 🤓

**Review by khanhnwin — APPROVED** [review]

**Pull request merged**